### PR TITLE
Enumerate search result index from 1

### PR DESCRIPTION
**Pull request by tintinthong**
https://linear.app/cardstack/issue/CS-7106/enumerate-ai-assistant-search-with-1-based-index

Screenshot 2024-08-29 at 16 34 10

### CODE REVIEW

File: packages/base/command-result.gts [hunk 1]
```diff
@@ -19,6 +19,7 @@ import {
 } from '@cardstack/boxel-ui/icons';
 import { Button, Header, IconButton } from '@cardstack/boxel-ui/components';
 import { on } from '@ember/modifier';
+import helper from '@ember/component/helper';
```
Comment (backspace) [inline packages/base/command-result.gts]:
```suggestion
```

This is the source of a lint error.

File: packages/base/command-result.gts [hunk 2]
```diff
@@ -222,6 +226,10 @@ class CommandResultEmbeddedView extends Component<typeof CommandResult> {
   </template>
 }
 
+export function add(value1: number, value2: number) {
```
Comment (backspace) [inline packages/base/command-result.gts]: Does it need to be exported?

File: packages/base/command-result.gts [hunk 3]
```diff
@@ -129,17 +130,20 @@ class CommandResultEmbeddedView extends Component<typeof CommandResult> {
...
-              {{/let}}
-            </div>
+            {{#let (add i 1) as |idx|}}
+              <div
+                class='card-item'
+                data-test-result-card={{cardResource.card.id}}
+                data-test-result-card-idx={{idx}}
+              >
+                {{#let (getComponent cardResource.card) as |Component|}}
+                  {{idx}}.
```
Comment (backspace) [inline packages/base/command-result.gts]: This works, though I don’t see any test assertions changing though for the `data-test-result-card-idx=`, could we use `<ol>` instead?

File: packages/base/command-result.gts [hunk 4]
```diff
@@ -222,6 +226,10 @@ class CommandResultEmbeddedView extends Component<typeof CommandResult> {
   </template>
 }
 
+export function add(value1: number, value2: number) {
+  return value1 + value2;
```
Comment (jurgenwerk) [inline packages/base/command-result.gts]: Just a heads up - I'm adding this as a math helper in https://github.com/cardstack/boxel/pull/1521/files#diff-89ed62e9ed0e7ef8d23a884faf267c8d41ec9c66af01e6c313efb8df65faa511

**Comment by habdelra** [comment]
@tintinthong I merged upstream, the CI host test failures that exist are real errors related to this update

File: packages/base/command-result.gts (hunk 3 above)
Comment (tintinthong) [reply packages/base/command-result.gts]: Thanks. I did this and removed idx all together

File: packages/base/command-result.gts (hunk 4 above)
Comment (tintinthong) [reply packages/base/command-result.gts]: I removed data-test-result-card-idx and used ol so I have no need for the math-helper. Thanks for the heads up

**Review by habdelra — APPROVED** [review]

**Pull request merged**